src/address.ts

```diff
@@ -182,6 +188,10 @@ export class Address {
      * Converts the address to a pretty, plain JavaScript object.
      */
     toJSON(): object {
+        if (this.isEmpty()) {
```

Comment:
Maybe we don't need this? I think it would be better to have the `bech32` and `pubkey` properties - they will be empty with the rest of the changes. Better than users getting `cannot read property of undefined` errors

Reply:
You are right, missed that :rocket:
Fixed :)